system/lzma/LzmaEnc.cpp

```diff
@@ -819,8 +819,7 @@ static void LenEnc_Encode2(CLenPriceEnc *p, CRangeEnc *rc, UInt32 symbol, UInt32
 }
 
 
-
-
+static void MovePos(CLzmaEnc *p, UInt32 num) __attribute__((no_sanitize("function")));
```

Comment:
These uses of __attribute__ fail to compile on windows - probably because platform.h isn't included.

The [[ syntax ]] is supported, but gives a warning (which could be suppressed):

`static void MovePos(CLzmaEnc *p, UInt32 num) [[no_sanitize("function")]];
`
I've verified it works on 2015 and 2017.

Reply:
```
[[no_sanitize("function")]]
static void MovePos(CLzmaEnc *p, UInt32 num)
{
}

```
also works.